kernel/Kconfig

```
@@ -900,6 +900,15 @@ config STACK_CANARIES_STRONG
 	  functions that have local array definitiion or have references to local
 	  frame addresses.
 
+config STACK_CANARIES_EXPLICIT
+	bool "Compiler stack canaries in explicit functions"
```

related to other Kconfig choice entry prompt proposal:

```suggestion
	bool "Explicit functions"
```
Only a proposal, thus non-blocking .